src/libraries/System.Reflection.Metadata/src/System/Reflection/Metadata/Ecma335/Encoding/ControlFlowBuilder.cs

```diff
@@ -75,14 +86,14 @@ internal int GetBranchDistance(ImmutableArray<int>.Builder labels, ILOpCode bran
             }
         }
 
-        private readonly ImmutableArray<BranchInfo>.Builder _branches;
-        private readonly ImmutableArray<int>.Builder _labels;
-        private ImmutableArray<ExceptionHandlerInfo>.Builder? _lazyExceptionHandlers;
+        private readonly List<BranchInfo> _branches;
+        private readonly List<int> _labels;
+        private List<ExceptionHandlerInfo>? _lazyExceptionHandlers;
```

Comment:
What is the reason of using List instead of  ImmutableArray builder

Reply:
We are not going to build an immutable array.